raiden_contracts/utils/token_ops.py

```diff
@@ -0,0 +1,183 @@
+import functools
+import click
+import requests
+from eth_utils import to_checksum_address
+from web3 import HTTPProvider, Web3
+from web3.middleware import construct_sign_and_send_raw_middleware, geth_poa_middleware
+from raiden_contracts.constants import CONTRACT_CUSTOM_TOKEN
+from raiden_contracts.contract_manager import ContractManager, contracts_precompiled_path
+from raiden_contracts.utils.private_key import get_private_key
+from raiden_contracts.utils.signature import private_key_to_address
+from raiden_contracts.utils.transaction import check_successful_tx
+
+
+class TokenOperations:
+    def __init__(self, rpc_url: str, private_key: str, wait: int = 10):
```

Comment:
I would suggest adding a possible password file as well. What do you think?

Reply:
Sure let me add it.